src/Utils.php

```diff
@@ -921,4 +921,58 @@ public static function is_canary_island( string $state, string $country ): bool
 
 		return false;
 	}
+
+	/**
+	 * Get the frontend locations.
+	 *
+	 * @return array $locations
+	 */
+	public static function get_frontend_locations(): array {
+		$locations = array(
+			'cart_before_checkout'             => array(
+				'woocommerce_proceed_to_checkout',
+				'postnl_before_woocommerce/proceed-to-checkout-block',
+			),
+			'cart_after_checkout'              => array(
+				'woocommerce_after_cart_totals',
+				'postnl_after_woocommerce/proceed-to-checkout-block',
+			),
+			'checkout_before_customer_details' => array(
+				'woocommerce_checkout_before_customer_details',
+			),
+			'checkout_after_customer_details'  => array(
+				'woocommerce_checkout_after_customer_details',
+			),
+			'minicart_before_buttons'          => array(
+				'woocommerce_widget_shopping_cart_before_buttons',
+				'postnl_before_woocommerce/mini-cart-footer-block',
+			),
+			'minicart_after_buttons'           => array(
+				'woocommerce_widget_shopping_cart_after_buttons',
+				'postnl_after_woocommerce/mini-cart-footer-block',
+			),
+		);
+		// Allow filtering of the locations.
+		return apply_filters( 'postnl_frontend_locations', $locations );
```

Comment:
```suggestion
		// Allow filtering of the locations.
		return apply_filters(
		'postnl_frontend_locations',
		array(
			'cart_before_checkout'             => array(
				'woocommerce_proceed_to_checkout',
				'postnl_before_woocommerce/proceed-to-checkout-block',
			),
			'cart_after_checkout'              => array(
				'woocommerce_after_cart_totals',
				'postnl_after_woocommerce/proceed-to-checkout-block',
			),
			'checkout_before_customer_details' => array(
				'woocommerce_checkout_before_customer_details',
			),
			'checkout_after_customer_details'  => array(
				'woocommerce_checkout_after_customer_details',
			),
			'minicart_before_buttons'          => array(
				'woocommerce_widget_shopping_cart_before_buttons',
				'postnl_before_woocommerce/mini-cart-footer-block',
			),
			'minicart_after_buttons'           => array(
				'woocommerce_widget_shopping_cart_after_buttons',
				'postnl_after_woocommerce/mini-cart-footer-block',
			),
		)
		);
```
We can shorten it to be like the code above.